homeworks/07-mars-viewer/src/Types/state.ts

```diff
@@ -0,0 +1,13 @@
+export enum MarsViewerTab {
+    CHOOSE_SOLUTION = 'CHOOSE_SOLUTION',
+    FAVOURITE_PHOTOS = 'FAVOURITES'
+}
+
+export interface MarsViewerState {
+    currentSelectedSolution: number
+    loading: boolean
+    loaded: boolean
```

Comment:
this variable can be computed out of `loading` and `imagesLinks.length` values, storing it in separate place in state may lead to bugs with additional features, and does not comply with the concept of the single source of truth